_data/simple-icons.json

```diff
@@ -2168,6 +2168,11 @@
             "source": "https://www.datadoghq.com/about/resources",
             "guidelines": "https://www.datadoghq.com/about/resources/"
         },
+        {
+            "title": "Dataiku",
+            "hex": "00B2A9",
```

Comment:
Out of curiosity, how did you choose this color?

On their website, they often use `#2ab1ac` and I found `#2ab6b9` embedded in their teal logo.

Reply:
I'm not exactly sure from where I got that color from, since it's been a while since I made that commit 😅
I'll change the mentioned hex to `#2ab1ac`